spring-security/src/main/java/com/epam/esm/user/model/UserDTOBuilder.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.esm.user.model;
+
+import com.epam.esm.user.dto.UserDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+@Slf4j
+@Component
+public class UserDTOBuilder {
+
+    public UserDTO convertUserToUserDTO(User user) {
```

Comment:
By the logic of this class it looks more like mapper to me

Reply:
I think so . I remade ,ty